tempesta_fw/http_parser.c

```diff
@@ -3195,7 +3195,8 @@ tfw_http_parse_terminate(TfwHttpMsg *hm)
 	BUG_ON(!hm);
 	BUG_ON(!(TFW_CONN_TYPE(hm->conn) & Conn_Srv));
 
-	if (hm->parser.state == Resp_BodyUnlimRead) {
+	if (hm->parser.state == Resp_BodyUnlimRead
+	    || hm->parser.state == Resp_BodyUnlimStart) {
```

Comment:
In a complex and lengthy `if` statements it's difficult then to separate the body of the `if` block from the conditions. For that reason the opening curly brace if moved to the next, separate line.